services_app/src/main/java/org/opendatakit/services/sync/service/logic/AggregateSynchronizer.java

```diff
@@ -1053,14 +1053,20 @@ public void uploadInstanceFileBatch(List<CommonFileAttachmentTerms> batch,
   @Override
   public void downloadInstanceFileBatch(List<CommonFileAttachmentTerms> filesToDownload,
       String serverInstanceFileUri, String instanceId, String tableId) throws HttpClientWebException, IOException {
-    // boolean downloadedAllFiles = true;
+    downloadInstanceFileBatch(filesToDownload, serverInstanceFileUri, instanceId, tableId, false);
+  }
 
+  @Override
+  public void downloadInstanceFileBatch(List<CommonFileAttachmentTerms> filesToDownload,
+      String serverInstanceFileUri, String instanceId, String tableId, boolean reduce) throws HttpClientWebException, IOException {
+    // boolean downloadedAllFiles = true;
     URI instanceFilesDownloadUri = wrapper.constructInstanceFileBulkDownloadUri(serverInstanceFileUri, instanceId);
 
     ArrayList<OdkTablesFileManifestEntry> entries = new ArrayList<OdkTablesFileManifestEntry>();
     for (CommonFileAttachmentTerms cat : filesToDownload) {
       OdkTablesFileManifestEntry entry = new OdkTablesFileManifestEntry();
```

Comment:
Also, OdkTablesFileManifestEntry is one of the classes found in the odk-rest-interface jar.  In order for this change to work, the odk-rest-interface jar will need to be packaged with the change to OdkTablesManifestEntry in the sync-endpoint and put into androidlibrary under the androidlibrary/androidlibrary_lib/libs directory.